go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/code-ready/crc
 require (
 	github.com/BurntSushi/toml v0.3.1 // indirect
 	github.com/DATA-DOG/godog v0.7.13
+	github.com/Masterminds/semver v1.5.0
```

Comment:
Wouldn't https://github.com/coreos/go-semver make more sense for us?

Reply:
@cfergeau that is outdated one, no development from last 2 years :)